scripts/convert-svgs-to-icon-mixins.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs');
+const path = require('path');
+const mime = require('mime');
+const SVGO = require('svgo');
+
+const args = process.argv;
+const symbols = /[\r\n%#()<>?\[\\\]^`{|}]/g;
+const svgo = new SVGO();
+
+function encodeSVG(data) {
+  // Use single quotes instead of double to avoid encoding.
+  data = data.replace(/"/g, "'");
+
+  data = data.replace(/>\s{1,}</g, '><');
+  data = data.replace(/\s{2,}/g, ' ');
```

Comment:
I know these 2 lines come from my initial script, and I try to recall why they are here.
They seem to remove whitespace, but I would guess that SVGO should do that as well?

Can you verify if these lines are needed? Is the output different with/without them?

Reply:
If they are needed for some reason, please add a one line comment above them to explain a bit why we need those.

Reply:
Oh true - it worked, so didn't think anything of these ;) They're gone.